### PR TITLE
docs: redis sink docs

### PR DESCRIPTION
**Pull request by sumitaich1998**

### CODE REVIEW

File: docs/sinks/redis.md [hunk 1]
```diff
@@ -0,0 +1,16 @@
+# Redis Sink
+
+### Input Schema
+Redis  sink in Depot only supports Protobuf schema as of now. Support for JSON and Avro format is planned for a future release of Depot.
```
Comment (lavkesh) [inline docs/sinks/redis.md:4]: It supports json too :)

File: docs/sinks/redis.md [hunk 2]
```diff
@@ -0,0 +1,16 @@
...
+
+### Data Types
+Redis sink can be created in 2 different modes based on the value of [`SINK_REDIS_DATA_TYPE`](../sinks/redis-sink.md#sink_redis_data_type): HashSet or List
+  - `Hashset`: For each message, an entry of the format `key : field : value` is generated and pushed to Redis. field and value are generated on the basis of the config [`INPUT_SCHEMA_PROTO_TO_COLUMN_MAPPING`](../sinks/redis-sink.md#-input_schema_proto_to_column_mapping-2)
+  - `List`: For each message entry of the format `key : value` is generated and pushed to Redis. Value is fetched for the proto index provided in the config [`SINK_REDIS_LIST_DATA_PROTO_INDEX`](../sinks/redis-sink.md#sink_redis_list_data_proto_index)
+
+The `key` is picked up from a field in the message itself.
+
+Limitation: Depot Redis sink only supports HashSet and List entries as of now.
```
Comment (lavkesh) [inline docs/sinks/redis.md:13]: It supports Key-value, list and hashset.

File: docs/sinks/redis.md [hunk 3]
```diff
@@ -0,0 +1,16 @@
+# Redis Sink
+
+### Input Schema
+Redis  sink in Depot only supports Protobuf schema as of now. Support for JSON and Avro format is planned for a future release of Depot.
+
+### Data Types
+Redis sink can be created in 2 different modes based on the value of [`SINK_REDIS_DATA_TYPE`](../sinks/redis-sink.md#sink_redis_data_type): HashSet or List
+  - `Hashset`: For each message, an entry of the format `key : field : value` is generated and pushed to Redis. field and value are generated on the basis of the config [`INPUT_SCHEMA_PROTO_TO_COLUMN_MAPPING`](../sinks/redis-sink.md#-input_schema_proto_to_column_mapping-2)
```
Comment (lavkesh) [inline docs/sinks/redis.md:8]: `INPUT_SCHEMA_PROTO_TO_COLUMN_MAPPING` this config is not present. 
can you rephrase it?

Comment (sumitaich1998) [reply docs/sinks/redis.md]: done

File: docs/sinks/redis.md [hunk 4]
```diff
@@ -0,0 +1,17 @@
+# Redis Sink
+
+### Input Schema
+Redis  sink in Depot only supports Protobuf and JSON schema as of now. Support for Avro format is planned for a future release of Depot.
+
+### Data Types
+Redis sink can be created in 3 different modes based on the value of [`SINK_REDIS_DATA_TYPE`](../reference/configuration/redis.md#sink_redis_data_type): HashSet, KeyValue or List
+  - `Hashset`: For each message, an entry of the format `key : field : value` is generated and pushed to Redis. Field and value are generated on the basis of the config [`SINK_REDIS_HASHSET_FIELD_TO_COLUMN_MAPPING`](../reference/configuration/redis.md#sink_redis_hashset_field_to_column_mapping)
+  - `List`: For each message entry of the format `key : value` is generated and pushed to Redis. Value is fetched for the proto index provided in the config [`SINK_REDIS_LIST_DATA_PROTO_INDEX`](../reference/configuration/redis.md#sink_redis_list_data_proto_index) or the Proto field name provided in the config [`SINK_REDIS_LIST_DATA_FIELD_NAME`](../reference/configuration/redis.md#sink_redis_list_data_field_name)
```
Comment (lavkesh) [inline docs/sinks/redis.md:9]: ah. we need to delete `SINK_REDIS_LIST_DATA_PROTO_INDEX` it's not required.

Comment (lavkesh) [reply docs/sinks/redis.md:9]: `SINK_REDIS_LIST_DATA_FIELD_NAME` this is the config.

**Review by lavkesh — APPROVED** [review]
